charts/jaeger/values.yaml

```diff
@@ -16,7 +16,7 @@ fullnameOverride: ""
 allInOne:
   enabled: false
   image: jaegertracing/all-in-one
-  tag: 1.29.0
+  tag: 1.37.0
```

Comment:
Can we just remove this instead?  That will be consistent with the rest of the chart, which defaults image tags to the chart's appVersion

Reply:
Why not using the tag "latest" ?